### ISSUE TITLE
Neovim: Suggestions and hover are broken for the latest update

### ISSUE THREAD

**Issue by zackAJ** [tailwindlabs/tailwindcss-intellisense#1034]
**What version of VS Code are you using?**

NVIM v0.10.0

**What version of Tailwind CSS IntelliSense are you using?**

v0.12.7

**What version of Tailwind CSS are you using?**

v3.2.1

**What package manager are you using?**

npm (via Mason) v0.0.23

**What operating system are you using?**

Linux

**Tailwind config**

```js
import defaultTheme from 'tailwindcss/defaultTheme';
import forms from '@tailwindcss/forms';

/** @type {import('tailwindcss').Config} */
export default {
  important: true,
  content: [
    `${process.env.PWD}/index.html`,
    `${process.env.PWD}/src/**/*.{vue,js,ts,jsx,tsx}`,
    `${process.env.PWD}/vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php`,
    `${process.env.PWD}/storage/framework/views/*.php`,
    `${process.env.PWD}/resources/views/**/*.blade.php`,
  ],

  theme: {
    extend: {
      fontFamily: {
        sans: ['Figtree', ...defaultTheme.fontFamily.sans],
      },
      colors: {
        "primary" : "#1976D2",
        "secondary" : "#002FA6",
        "accent" : "#9C27B0",
        "dark" : "#1D1D1D",
        "dark-page" : "#121212",
        "positive" : "#21BA45",
        "negative" : "#C10015",
        "info" : "#31CCEC",
        "warning" : "#F2C037",
      }
    },
  },

  plugins: process.env.PWD.includes("front") ? [] : [forms]
};

```

**Neovim settings**

```lua
    tailwindcss = { filetypes = { "html", "blade", "vue" } },
```

**Reproduction URL**
None

**Describe your issue**

I get no suggestions and no hover, client is attached to the buffer.
![image](https://github.com/user-attachments/assets/20bcd8ee-0547-4bef-bb98-fa26f8b1d9cf)
when downgrading to the previous version it works fine

**Comment by zackAJ** [comment]
for Mason users downgrade using this command until it's fixed:
![image](https://github.com/user-attachments/assets/87c9f171-3be4-40ad-82cb-55b24207ca34)
```
:MasonInstall tailwindcss-language-server@0.0.22
```

**Comment by philipp-spiess** [comment]
Hey @joy-stacks! Thanks for reporting this and sorry for the troubles! We just released v0.0.24 of the language server that resolves this issue. Please let us know if there are still issues.
